server/odc-migrate/src/main/resources/migrate/common/V_4_3_0_4__add_unique_identifier.sql

```diff
@@ -1 +1 @@
-ALTER TABLE `collaboration_project` ADD COLUMN `unique_identifier` VARCHAR(128) DEFAULT NULL COMMENT 'Unique identifier, from external system';
\ No newline at end of file
+ALTER TABLE `collaboration_project` ADD COLUMN `unique_identifier` VARCHAR(128) NOT NULL COMMENT 'Unique identifier, may from external system or UUID generated by ODC if no external identifier';
```

Comment:
how about simpliy use 

`update collaboration_project set unique_identifier=concat('ODC_', uuid()) where unique_identifier is null;`

Reply:
ok~